### ISSUE TITLE
Parse latex as input to Unit

### ISSUE THREAD

**Issue by nstarman** [astropy/astropy#11598]
E.g. it would be cool if `` u.Unit(r"$\frac{km}{s}$") `` could be correctly parsed as km/s.

**Comment by pllim** [comment]
But why?

**Comment by nstarman** [comment]
Mostly for symmetry. Units can be output as latex, but not input.

https://docs.astropy.org/en/stable/_modules/astropy/units/format/latex.html?highlight=latex

**Comment by pllim** [comment]
Does symmetry alone justify the work? Is there a real need for this feature? Do people actually try to turn LaTeX string into `astropy` units? 🤔

**Comment by nstarman** [comment]
It might be more effort than it's worth. I encountered this in #11595, where I proposed unifying the definition of physical types with the documentation using a CSV file. In there, having Unit parse latex would mean the auto-generated documentation would look good — `$m^2$` over `m2`... But it looks like #11595 might not use that method at all, making this irrelevant. Can we keep open until #11595 is resolved. Thanks!
